pages/samvera/manager_guide/hyrax_2.1/batch-ops-2.1.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Batch Operations"
+permalink: batch-ops-2.1.html
+folder: samvera/manager_guide/batch-works-2.1.md
+sidebar: home_sidebar
+a-z: ['Batch Operations - Repository Administration']
+keywords: Best Practices, managers, repo mangers, hyrax administration
+tags: [feature_guide]
+categories: How to use the Administration panel in hyrax
+version:
+  id: 'hyrax_2.1-stable'
+  versions:
+  - label: 'Hyrax 1.0'
+    link:  'batch-works-1.0.html'
+  - label: 'Hyrax 2.0'
+    link:  'batch-works-2.0.html'
+  - label: 'Hyrax 2.1'
+    link:  'batch-ops-2.1.html'
+    selected: 'true'
+---
+## Uploading Works
+Batch upload lets you upload multiple files at once as separate works. This is useful when you have several works, where each work is a single file, and each work has the same creator, such as technical reports from a organization or photographs by an artist.
```

Comment:
can we change this to say "...and each work has the same descriptive metadata, such as technical reports from a organization or photographs by an artist."

Reply:
yep